app/controllers/api_controller/parameters.rb

```diff
@@ -114,6 +114,20 @@ def expand?(what)
       expand_param ? expand_param.include?(what.to_s) : false
     end
 
+    def decorator_selection
+      if params['decorators']
```

Comment:
It's not, I just copied the `attriubutes_selection` pattern really.. simplified :) thanks